File: 이상한 문자 만들기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
    string answer = "";
    int ch = 0;
    for (int i = 0; i < s.size(); i++) {
        if (s[i] == ' ') {
            ch = 0;
            continue;
        }
        if (ch % 2 == 0 && (s[i] >= 'a' && s[i] <= 'z')) {
            s[i] -= 32;
        }
        else if (ch % 2 == 1 && (s[i] >= 'A' && s[i] <= 'Z')) {
            s[i] += 32;
        }
        ch++;
    }
    answer = s;
    return answer;
}

File: Lv.1/Python/이상한 문자 만들기.py
def solution(s):
    answer= s.split(' ')
    a =[]
    for i in range(len(answer)):
        for j in range(len(answer[i])):
            if j % 2 == 0:
                a.append(answer[i][j].upper())
            else:a.append(answer[i][j].lower())
        a.append(' ')
    del a[-1]
    return "".join(a)